### ISSUE TITLE
No README.md

### ISSUE THREAD

**Issue by lylet-AC** [shafferz/caster-2d#1]
Please add a README.md for understanding the installation of requirements, setup of the program, and how to run / test the program

**Comment by shafferz** [comment]
Good call, I just haven't written a README yet due to the nature of the project (read: nobody is supposed to install and run the program on the user end, as the current stage of development does not support a stable public release.)
